File: ZkTony-Manager/app/src/main/kotlin/com/zktony/manager/data/remote/model/Application.kt
package com.zktony.manager.data.remote.model

data class Application(
    val id: Int,
    val application_id: String,
    val build_type: String,
    val download_url: String,
    val version_name: String,
    val version_code: Int,
    val description: String,
    val create_time: String,
)

File: Rust/App/entity/src/application.rs
use sea_orm::entity::prelude::*;

#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
#[sea_orm(table_name = "application")]
pub struct Model {
    #[sea_orm(primary_key, auto_increment = false)]
    pub id: i32,
    pub application_id: String,
    pub build_type: String,
    pub download_url: String,
    pub version_name: String,
    pub version_code: i32,
    #[sea_orm(column_type = "Text")]
    pub description: String,
    pub create_time: Option<DateTime>,
}

#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {}

impl ActiveModelBehavior for ActiveModel {}
